PrebidMobile/PrebidServerAdapter/src/main/java/org/prebid/mobile/prebidserver/internal/Settings.java

```diff
@@ -15,7 +15,7 @@
     // connection settings
     public static final String REQUEST_URL_NON_SECURE = "http://prebid.adnxs.com/pbs/v1/auction";
     public static final String REQUEST_URL_SECURE = "https://prebid.adnxs.com/pbs/v1/auction";
-    public static final int REQUEST_TIME_OUR_MILLIS = 500;
+    public static int REQUEST_TIME_OUR_MILLIS = 500;
```

Comment:
If this can be set now, does it still make sense to have it as a constant?

I think if we're going to expose setting the time out in the client facing API, we should write a setter method and make sure its documented, rather than just removing the final here.